docs/user_manual.md

```diff
@@ -500,3 +501,14 @@ You can build this kind of configuration easily from the sample deployment by ch
 * Delete the configuration for the `WATCH_NAMESPACE` variable
 * Change the Roles to ClusterRoles
 * Change the RoleBindings to ClusterRoleBindings
+
+# Renaming a Cluster
+
+The name of a cluster is immutable, and it is included in the names of all of the dependent resources, as well as in labels on the resources. If you want to change the name later on, you can do so with the following steps. This example assumes you are renaming the cluster `sample-cluster` to `sample-cluster-2`.
+
+1.  Create a new cluster named `sample-cluster-2`, using the current connection string from `sample-cluster-2` as its seedConnectionString. You will need to set an `instanceIdPrefix` for `sample-cluster-2` to be different from the `instanceIdPrefix` for `sample-cluster`, to make sure that the instance IDs do not collide. The rest of the spec can match `sample-cluster`, other than any fields that you want to customize based on the new name. Wait for reconciliation on `sample-cluster-2` to complete.
```

Comment:
Just double-checking that we want to use the new connection string as the seedConnectionString?

Reply:
indeed, we do not